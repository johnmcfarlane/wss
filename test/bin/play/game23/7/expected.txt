 42  6,10 - DRIFT
 36  6,10 - THIRD
 22  6,1  | BIRTHS
 22  2,2  | BRIDAL
 22  6,10 - DRIB
 22  6,1  | FIRTHS
 22  6,1  | FRITHS
 22  6,10 - RFB
 20  2,10 - BE
 20  2,10 | BO
 20  6,10 - DFT
 20  6,10 - DIB
 20  6,10 - DIF
 20  6,1  | DRIFTS
 20  6,10 - HID
 20  6,10 - RHB
 20  2,2  | TRIBAL
 18  6,2  | BIRDS
 18  6,2  | DRIBS
 18  6,10 - HIR
 18  6,10 - HIT
 18  6,10 - RIFT
 18  6,1  | THIRDS
 18  6,10 - TRIB
 16  6,2  | BRITS
 16  6,10 - DIRT
 16  6,2  | FRITS
 16  2,10 - HE
 16  6,10 - HI
 16  2,10 | HO
 16  6,10 - HR
 16  6,10 - HT
 16  6,10 - RHD
 16  6,10 - RIB
 16  6,2  | RIFTS
 16  6,10 - TFR
 16  6,10 - THIR
 16  6,10 - TIB
 16  6,10 - TRF
 14  6,10 - DIR
 14  6,10 - DIT
 14  6,10 - RF
 14  6,10 - TRID
 12  2,10 - DE
 12  6,10 - DI
 12  6,2  | DIRTS
 12  2,10 | DO
 12  6,10 - RID
 12  6,10 - TDR
 12  6,10 - TH
 12  6,10 - TID
 10  2,5  | BALI
 10  6,3  | FIBS
 10  6,10 - RIT
 10  6,10 - RTI
 10  6,10 - TD
 10  6,10 - TRI
  8  6,3  | BIDS
  8  2,5  | DALI
  8  2,4  | DHAL
  8  6,3  | DIBS
  8  6,3  | DIFS
  8  6,3  | FIDS
  8  7,7  - ITD
  8  2,10 - TE
  8  6,10 - TI
  8  2,10 | TO
  8  6,8  | mEH
  7  2,5  | BAL
  7  6,4  | BDS
  7  6,3  | BITS
  7  2,3  | DITAL
  7  6,3  | FIRS
  7  6,3  | FITS
  7  6,3  | FRIS
  7  6,5  - IB
  7  7,7  - IB
  7  6,5  - IF
  7  7,7  - IF
  7  6,3  | RIBS
  7  6,3  | RIFS
  7  2,5  | TALI
  7  2,3  | TIDAL
  6  6,4  | BIS
  6  2,4  | DIAL
  6  6,4  | FRS
  6  2,5  | HAL
  6  6,3  | HITS
  6  6,4  | IFS
  6  6,5  - ITD
  6  6,4  | RFS
  6  6,4  | TBS
  6  6,3  | THIS
  6  2,3  | TRIAL
  6  6,8  | mED
  5  2,5  | DAL
  5  6,3  | DITS
  5  6,4  | HIS
  5  6,4  | HRS
  5  6,4  | HTS
  5  6,5  - ID
  5  7,7  - ID
  5  6,4  | IHS
  5  2,4  | ITAL
  5  2,4  | RIAL
  5  6,3  | RIDS
  4  6,4  | DIS
  4  6,4  | IDS
  4  6,5  - IT
  4  7,7  - IT
  4  2,5  | TAL
  4  6,8  | mER
  4  6,8  | mET
  3  6,4  | IRS
  3  6,4  | ITS
  3  6,4  | TIS
  3  6,4  | TRS
  2  6,5  | IS
  2  7,7  | IS
