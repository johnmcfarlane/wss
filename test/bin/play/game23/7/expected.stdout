 42  6,10 - DRIFT
 36  6,10 - THIRD
 22  6,1  | BIRTHS
 22  2,2  | BRIDAL
 22  6,1  | BRITHS
 22  6,10 - DRIB
 22  6,1  | FIRTHS
 22  6,1  | FRITHS
 22  5,10 - ID
 20  2,10 - BE
 20  2,10 | BO
 20  6,10 - DIB
 20  6,10 - DIF
 20  6,1  | DRIFTS
 20  2,2  | TRIBAL
 18  6,2  | BIRDS
 18  6,2  | DRIBS
 18  5,10 - IT
 18  6,1  | THIRDS
 16  6,2  | BRITS
 16  6,10 - DIRT
 16  6,2  | FRITS
 16  2,10 - HE
 16  2,10 | HO
 16  6,2  | RIFTS
 16  6,10 - THIR
 14  6,10 - DIT
 12  2,10 - DE
 12  6,2  | DIRTS
 12  2,10 | DO
 10  6,3  | FIBS
 10  5,6  | NEWSIE
  8  6,3  | BIDS
  8  2,4  | DHAL
  8  6,3  | DIBS
  8  6,3  | DIFS
  8  6,3  | FIDS
  8  6,10 - TI
  7  2,5  | BAL
  7  6,3  | BITS
  7  6,3  | BRIS
  7  6,3  | FIRS
  7  6,3  | FITS
  7  6,5  - IB
  7  7,7  - IB
  7  6,5  - IF
  7  7,7  - IF
  7  6,3  | RIBS
  7  6,3  | RIFS
  7  2,5  | TALI
  7  2,3  | TIDAL
  6  6,4  | BIS
  6  2,4  | DIAL
  6  6,3  | HITS
  6  6,4  | IFS
  6  6,3  | THIS
  6  2,3  | TRIAL
  6  6,8  | mED
  5  2,5  | DAL
  5  6,3  | DITS
  5  6,4  | HIS
  5  6,5  - ID
  5  7,7  - ID
  5  2,4  | RIAL
  5  6,3  | RIDS
  4  6,4  | DIS
  4  6,4  | IDS
  4  6,5  - IT
  4  7,7  - IT
  4  6,8  | mET
  3  6,4  | ITS
  3  6,4  | TIS
  2  6,5  | IS
  2  7,7  | IS
